### DISCUSSION TITLE
GRPC: support metadata in the response

### DISCUSSION THREAD

**Discussion by ziomill** [Kong/insomnia#5602] [Ideas]
Hi guys,

To this day it seems that with Insomnia it is not possible to see the metadata attached to a GRPC response.
In my opinion, it could be a very useful capability. 

Is there a chance that Insomnia will support response metadata in the next releases?

thank you

**Comment by banks117** [comment]
Hey @ziomill! This is actually supported just through the Headers tab. I am working on something using gRPC metadata and was testing it.
